### DISCUSSION TITLE
I created a PostgreSQL wrapper to load test databases

### DISCUSSION THREAD

**Discussion by guel-codes** [locustio/locust#2835] [Show and tell]
## Overview
Hey everyone, I recently created this PostgreSQL client to load test some databases. I would like to add it to the examples directory just in case someone needs it in the future, they can find it right at the source. I tried to create a PR but for some reason it did allow me to. 

The code can be found here:
https://github.com/guel-codes/db_loadtesting
in the `loadtest.py` file. 

### Future Work:
I also want to create wrappers for the databases below at some point. So maybe we can create a `databases/` sub-directory inside of examples to store them all in there.

- Mongo
- MySQL
- Redis 

Let me know if you have any questions

**Comment by cyberw** [comment]
Looks amazing! You should definitely be able to make a PR. Just fork locust into your own github account and make a PR from your fork to the master branch of locustio/locust.

Traditionally I've directed this kind of contributions to the locust-plugins repo, but I might reconsider that further on anyway, so having it in examples/databases subdir makes sense, at least for now!

**Reply by guel-codes** [reply]
That would be awesome. Let me get a PR together and I will share it with everyone. Thanks again for the quick feedback!

**Comment by guel-codes** [comment]
@cyberw here is that PR. First time using ruff so it took a while to figure out some of the formatting issues and how to fix them. But everything should be good now.

https://github.com/locustio/locust/pull/2836

**Reply by cyberw** [reply]
Awesome! If you'll be contributing more stuff I recommend setting upp [pre-commit](https://pre-commit.com) so you'll see any errors before committing :)

**Reply by guel-codes** [reply]
Sounds good, I will get that setup as well. Thank you 🙏🏽
